platform/src/main/java/org/stellar/anchor/platform/IntegrationConfig.java

```diff
@@ -15,4 +17,10 @@ CustomerIntegration customerIntegration(Sep12Config sep12Config) {
     return new PlatformCustomerIntegration(
         sep12Config.getCustomerIntegrationEndPoint(), httpClient);
   }
+
+  @Bean
+  RateIntegration rateIntegration(Sep38Config sep38Config) {
+    OkHttpClient httpClient = OkHttpUtil.buildClient();
```

Comment:
💡 When I was implementing SEP-12's integration I was thinking that we should at some point update all our clients to use sensible timeout / retry / connection pool policies.

Not blocking for now though.

Reply:
That makes sense to me too. I created a bean for the HttpClient in https://github.com/stellar/java-stellar-anchor-sdk/pull/108/commits/e0741bd56fd7c894b13feb0231c2d498485e7935.